x11-misc/rofi-calc/rofi-calc-1.5.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils autotools
+
+DESCRIPTION="Do live calculations in rofi!"
+HOMEPAGE="https://github.com/svenstaro/rofi-calc"
+SRC_URI="https://github.com/svenstaro/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please open the `${PN}` to plaintext, makes copypasting easier and it's not going to change.